schema-forwarder/internal/transformer/transformer_test.go

```diff
@@ -69,17 +69,36 @@ func Test_SchemaTransformer_NoDataRetention(t *testing.T) {
 	})
 
 	t.Run("Test getSchemaKeyFromJob", func(t *testing.T) {
-		require.Equal(t, schemaTransformer.getSchemaKeyFromJob(testdata.TrackEvent, testdata.WriteKeyEnabled), &testdata.TestEventSchemaKey)
+		expectedKey := &proto.EventSchemaKey{
+			WriteKey:        testdata.WriteKeyEnabled,
+			EventType:       "track",
+			EventIdentifier: "event-name",
+		}
+		require.Equal(t, schemaTransformer.getSchemaKeyFromJob(testdata.TrackEvent, testdata.WriteKeyEnabled), expectedKey)
```

Comment:
why do we need any modification here?
```suggestion
		require.Equal(t, schemaTransformer.getSchemaKeyFromJob(testdata.TrackEvent, testdata.WriteKeyEnabled), &testdata.TestEventSchemaKey)
```